### PR TITLE
[Checkbox] Add chromatic story

### PR DESCRIPTION
**Pull request by benoitgrelard**

### CODE REVIEW

File: packages/react/checkbox/src/Checkbox.test.tsx [hunk 1]
```diff
@@ -47,52 +33,19 @@ describe('given a default Checkbox', () => {
...
-
-    it('should have a data-state attribute on the container set to `checked`', () => {
-      expect(checkbox).toHaveAttribute('data-state', 'checked');
-    });
-
-    it('should have a data-state attribute on the indicator set to `checked`', () => {
-      expect(indicator).toHaveAttribute('data-state', 'checked');
-    });
-
-    describe('and clicking the checkbox again', () => {
```
Comment (jjenzz) [inline packages/react/checkbox/src/Checkbox.test.tsx:64]: Do we want to keep this one somehow because it's testing interaction?

**Review by jjenzz — APPROVED** [review]
One question, otherwise :+1:

File: packages/react/checkbox/src/Checkbox.test.tsx (hunk 1 above)
Comment (benoitgrelard) [reply packages/react/checkbox/src/Checkbox.test.tsx:64]: I'll check this now.

Comment (benoitgrelard) [reply packages/react/checkbox/src/Checkbox.test.tsx:64]: Yeah good shout, I had remove it cause it was checking data-state, but I've rewritten to check for the indicator

**Review by jjenzz — APPROVED** [review]

**Pull request merged**